lib/SIL/SILVerifier.cpp

```diff
@@ -61,9 +61,16 @@ static llvm::cl::opt<bool> AbortOnFailure(
                               "verify-abort-on-failure",
                               llvm::cl::init(true));
 
+// SWIFT_ENABLE_TENSORFLOW
+// This flag is temporarily set to false because debug scope verification does
+// not handle inlined call sites. This is problematic for deabstraction, which
+// does performance inlining at -Onone.
+// When debug scope verification handles inlined call sites, set this flag to
+// true.
+// Documented at SR-8114.
 static llvm::cl::opt<bool> VerifyDIHoles(
                               "verify-di-holes",
-                              llvm::cl::init(true));
+                              llvm::cl::init(false));
```

Comment:
To reduce the amount of coverage loss, is it possible to only turn off this flag for S4TF related testing?

e.g. by setting the flag here https://github.com/apple/swift/blob/f79195c8e20043d83c184f26f5a384c6a73c3812/test/TensorFlow/integration.swift#L1.

Reply:
To clarify -- if this bug is expected to be fixed in upstream very soon, turning off this flag globally is fine.

Reply:
The scope of the "debug scope verification" problem isn't limited to testing: it's not reasonable to require users to specify `verify-di-holes=false` on every invocation of the compiler.

I think it makes sense to turn the flag off globally and to re-enable it when debug scope verification can handle inlining done during deabstraction.